docs/marionette.application.md

```diff
@@ -55,7 +55,7 @@ var RootView = require('./views/root');
 var App = Mn.Application.extend({
   region: '#root-element',
 
-  onStart: function() {
```

Comment:
This part isn't necessary as `app` and `options` aren't used here. I think it's fine to just leave the arguments set below and remove them here.